kafka-handler/src/java/org/apache/hadoop/hive/kafka/KafkaDagCredentialSupplier.java

```diff
@@ -114,7 +108,12 @@ private Token<?> getKafkaDelegationTokenForBrokers(Configuration conf, TableDesc
 
     Properties config = new Properties();
```

Comment:
In reference to refactoring variable name of copy in line no. 125 ., I think this name can be changed to **props or properties**. Improves code readability.

Reply:
In general, I like to keep changes to the minimal needed for the following reasons:
* facilitating reviews;
* minimizing merge conflicts;
* keeping git history/blame clean.

Indeed the name `props` or `properties` could be slightly better but doing the rename now would add necessary noise to this PR so I prefer to keep it as is.